src/closer-core.coffee

```diff
@@ -700,6 +700,26 @@ core =
         core.group_$_by(core.identity, coll))
      )
 
+  'not_$_every_$QMARK_' : (pred,coll) ->
+    assertions.arity 2, arguments.length
+    assertions.function pred
+    assertions.collection coll
+    !m.every? pred,coll
```

Comment:
This entire function can be reduced to `! core.every_$QMARK_ pred, coll` (even the assertions are handled by that function) :)